.gitlab-ci.yml

```diff
@@ -4,10 +4,11 @@ stages:
   
 variables:
   DATADOG_AGENT_WINBUILDIMAGES: v5996510-a9e6a7d
+  DRIVER_S3_BUCKET: dd-windowsfilter
```

Comment:
Why `dd-windowsfilter`? That seems like an odd name🤔 Also, I don't think this variable is actually used now?

Reply:
> `dd-windowsfilter`

Because this was the one used in this file before. We could create a new bucket as well indeed. I've no strong opinion tbh (besides my lazyness telling me this one is good enough).

>  I don't think this variable is actually used

I'll remove the variable, it brings no value.

Reply:
Done in 1ad8ac42a336fecd7aca60a84fe6b98aca0109d7. Let's discuss offline for the bucket

Reply:
I mean, I think it's a terrible name, but I also have little desire to be messing with S3 buckets, so "that's what it was" is good enough for me 😄